test/package/test_importer.py

```diff
@@ -84,6 +85,32 @@ def test_ordered_importer_basic(self):
             importer.import_module("package_a"),
         )
 
+    def test_ordered_importer_whichmodule(self):
+        """OrderedImporter's implementation of whichmodule should try each
+        underlying importer's whichmodule in order.
+        """
+
+        class DummyImporter(Importer):
+            def __init__(self, whichmodule_return):
+                self._whichmodule_return = whichmodule_return
+
+            def import_module(self, module_name):
+                raise NotImplementedError()
+
+            def whichmodule(self, obj, name):
+                return self._whichmodule_return
+
+        class DummyClass:
+            pass
+
+        dummy_importer_foo = DummyImporter("foo")
+        dummy_importer_bar = DummyImporter("bar")
+        foo_then_bar = OrderedImporter(dummy_importer_foo, dummy_importer_bar)
+        self.assertEqual(foo_then_bar.whichmodule(DummyClass(), ""), "foo")
+
+        bar_then_foo = OrderedImporter(dummy_importer_bar, dummy_importer_foo)
+        self.assertEqual(bar_then_foo.whichmodule(DummyClass(), ""), "bar")
```

Comment:
Consider adding a case in which the first importer fails to find the module that the object belongs to.